test_scripts/Polices/External_UCS/030_ATF_N_TC_LPT_Update_PreloadedPT_with_disallowed_by_external_consent_entities_on_struct_with_invalid_type_of_params.lua

```diff
@@ -0,0 +1,193 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT without "disallowed_by_external_consent_entities_on" struct
```

Comment:
@dboltovskyi, according to test set requirement should be "[Policies] External UCS: PreloadedPT with "disallowed_by_external_consent_entities_on" struct with invalid type of params"